extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -332,19 +332,25 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 	public get tableOfContentPaths() {
 		return this._tableOfContentPaths;
 	}
+
 	getUntitledNotebookUri(resource: string): vscode.Uri {
-		let title = this.findNextUntitledFileName(resource);
-		let untitledFileName: vscode.Uri = vscode.Uri.parse(`untitled:${title}`);
-		if (!this._allNotebooks.get(untitledFileName.fsPath)) {
+		let untitledFileName: vscode.Uri;
+		if (process.platform.indexOf('win') > -1) {
+			let title = path.join(path.dirname(resource), this.findNextUntitledFileName(resource));
+			untitledFileName = vscode.Uri.parse(`untitled:${title}`);
```

Comment:
>untitled: [](start = 40, length = 9)

Why not use with({ scheme: 'untitled' })?

Reply:
wish that works! I had that before. It only works on MAC that way but on windows it doesn't. the notebookUri is only preserved when I have it this way.